docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,183 @@
+
+# Setting up Service Catalog for API Aggregation in Kubernetes
+
+The aggregator is a server that sits in front of everything. 
```

Comment:
Also would be helpful if you had 1-2 sentences that motivate the need for an aggregator.  What problem does having an aggregator solve?